meta-ros2/recipes-devtools/ros-sdk-env/ros-sdk-env_1.0.bb

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Stephen Street stephen@redrocketcomputing.com
+
+SUMMARY = "Add ROS2 SDK enviroment variable"
+HOMEPAGE = "https://github.com/meta-ros"
+SECTION = "devel"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/ros-sdk-env:"
+
+SRC_URI += "file://generate-colcon-default"
```

Comment:
It looks like this file was not committed.

```
ERROR: /build/../work/src/ros/meta-ros2/recipes-devtools/ros-sdk-env/ros-sdk-env_1.0.bb: Unable to get checksum for nativesdk-ros-sdk-env SRC_URI entry generate-colcon-default: file could not be found
```

Reply:
Let me fix.